doc/sphinx/language/core/basic.rst

```diff
@@ -288,12 +288,14 @@ rest of the Coq manual: :term:`terms <term>` and :term:`types
      on terms (we say that the language is `dependently-typed
      <https://en.wikipedia.org/wiki/Dependent_type>`_).  Because of
      this, types and terms share a common syntax.  All types are terms,
-     but not all terms are types:
+     but not all terms are types.  (Some tactics use :n:`@one_type`,
+     which is a :n:`one_term` (syntactically) that is interpreted as a :n:`@type`.
```

Comment:
I find this parenthesis unsatisfying because it explains something about `one_type` specifically, but not about `type`. What about:
```suggestion
     but not all terms are types.  We define syntactic aliases :n:`@type`
     and :n:`@one_type` to clarify in the documentation when the grammar expects
     something that is syntactically of the class of :term:`terms <term>`, but
     semantically, it is a type that is expected:
```

Reply:
You like this?

```
     The syntactic aliases :n:`@type` and
     :n:`@one_type` are used to make clear when the provided :term:`term`
     must semantically be a type:
```